### PR TITLE
Update development instructions

### PR DESCRIPTION
**Pull request by celliott113**
Add step in Windows to ensure the user clones the project before proceeding onto next steps.

### CODE REVIEW

File: docs/DEVELOPMENT.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ Please follow the instructions below to be able to build the project from source
 <a name="windows"></a>
 ### Windows
 
+* You will need to clone the `fheroes2` project.
```
Comment (ihhub) [inline docs/DEVELOPMENT.md:10]: This is true not only on Windows but on all platforms.

**Review by ihhub — CHANGES_REQUESTED** [review]

File: docs/DEVELOPMENT.md (hunk 1 above)
Comment (celliott113) [reply docs/DEVELOPMENT.md]: You are correct, I have adjusted by removing from Windows and adding to the top of the doc to encompass all platforms.

File: docs/DEVELOPMENT.md [hunk 2]
```diff
@@ -4,6 +4,8 @@
 
 Please follow the instructions below to be able to build the project from source:
 
+<b>Note:</b> You will need to clone the `fheroes2` project before following the below instructions.
```
Comment (oleg-derevenetz) [inline docs/DEVELOPMENT.md:7]: There is no need to use HTML tags if there are corresponding Markdown tags:

```suggestion
**Note:** You will need to clone the `fheroes2` project before following the instructions below.
```

**Review by oleg-derevenetz — CHANGES_REQUESTED** [review]
Hi @celliott113 I just left one suggestion, please take a look when you have time.

**Review by celliott113 — COMMENTED** [review]
Updated with @oleg-derevenetz recommendations. Necessary to maintain document consistency within the markdown language.

**Review by oleg-derevenetz — APPROVED** [review]

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@celliott113 , thank you very much for your contribution to the project!

**Pull request merged**